### DISCUSSION TITLE
武汉大学 WHU

### DISCUSSION THREAD

**Discussion by StLeoX** [CollegesChat/university-information#118] [University]
生活环境一般
本科教学水平显然不及隔壁华科
本科教学内容整体偏旧，不过2021年有一波变化显著的课程改革

**Comment by afterexam** [comment]
听说食堂质量参差不齐吗？

**Reply by StLeoX** [reply]
目前看来就是` 桂园食堂 `稍微差一点，信息学部这边的食堂都还不错，也靠近街道口商圈可以去外面吃。

**Comment by StLeoX** [comment]
推荐项目 [OpenWHU](https://github.com/openwhu/OpenWHU)，整理了很多武大课程学习资料。

**Comment by wuduanq** [comment]
园区较大，务必购车，建议二手
